src/lib/drupal/lovell/tests/staticProps.test.ts

```diff
@@ -43,6 +83,90 @@ describe('getLovellStaticPropsContext', () => {
   })
 })
 
+describe('getLovellStaticPropsResource', () => {
+  describe('press release sorting', () => {
```

Comment:
So this is fine, but if you've only got one test inside this `describe()` and no need for an `afterEach()` or `afterAll()` clean up, you don't really need it wrapped in a second `describe()`. 🤷 

No harm to it, of course, but it's not necessary.